src/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -101,7 +101,6 @@ public void NotConnected_ThrowsException()
         public void SendFile_Synchronous(IPAddress listenAt, bool sendPreAndPostBuffers, int bytesToSend)
         {
             const int ListenBacklog = 1;
-            const int LingerTime = 10;
```

Comment:
There are lots of other instances of this pattern in the same file. If you believe this pattern isn't correct, are you proposing changing all our tests to avoid LingerOptions?

Before we start removing them, do we have other tests validating that LingerOptions work?